### PR TITLE
Feature/add r zulu

### PR DESCRIPTION
**Pull request by DrDano**
## add-recipe/login bug fixes/bug fixes
Develop was merged into this branch before pull request was made but please check all files for this one especially since it affects so many areas

* added add-recipe functionality and styling to form
* added styling to login form
* fixed issue where filters did not function in userprofile view
* Heather added some views as well which deal with editing recipes

### CODE REVIEW

File: models/Recipe.js [hunk 1]
```diff
@@ -99,4 +99,4 @@ Recipe.init({
 }
 );
 
-module.exports = Recipe;
```
Comment (DrDano) [inline models/Recipe.js:102]: This is a newline that was deleted.

File: models/Allergy.js [hunk 2]
```diff
@@ -63,4 +63,4 @@ Allergy.init({
 }
 );
 
-module.exports = Allergy;
```
Comment (DrDano) [inline models/Allergy.js:66]: This is a newline that was deleted.

File: package-lock.json [hunk 3]
```diff
@@ -5556,4 +5556,4 @@
       "integrity": "sha512-3wdGidZyq5PB084XLES5TpOSRA3wjXAlIWMhum2kRcv/41Sn2emQ0dycQW4uZXLejwKvg6EsvbdlVL+FYEct7A=="
     }
   }
-}
+}
```
Comment (DrDano) [inline package-lock.json:5559]: another newline

File: public/javascript/add-vote.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+async function voteClickHandler(event) {
```
Comment (DrDano) [inline public/javascript/add-vote.js:1]: This is code added by Heather before I branched from her branch

File: public/javascript/allergen-filter.js [hunk 5]
```diff
@@ -29,7 +29,7 @@ async function vegetarian() {
 }
```
Comment (DrDano) [inline public/javascript/allergen-filter.js:29]: This is the fix I'm adding to mitigate the issue where filters don't work on userprofile view

File: public/javascript/edit-post.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+async function editFormHandler(event) {
```
Comment (DrDano) [inline public/javascript/edit-post.js:1]: This is code added by heather before I branched off of her branch

File: public/javascript/logout.js [hunk 7]
```diff
@@ -11,8 +11,4 @@ async function logout() {
   }
```
Comment (DrDano) [inline public/javascript/logout.js:11]: Here I switch the logout from a button to an anchor that doesn't have an event listener and goes to the logout route

File: public/javascript/recipe-profile.js [hunk 8]
```diff
@@ -1,14 +1,14 @@
 async function getRecipe() {
```
Comment (DrDano) [inline public/javascript/recipe-profile.js:1]: all formatting changes

File: views/add-recipe.handlebars [hunk 9]
```diff
@@ -0,0 +1,37 @@
+<section class="container">
```
Comment (DrDano) [inline views/add-recipe.handlebars:1]: Code added by Heather before I branched off of her branch, but I added styling here.

File: views/edit-post.handlebars [hunk 10]
```diff
@@ -0,0 +1,17 @@
+<article>
```
Comment (DrDano) [inline views/edit-post.handlebars:1]: Code added by Heather before I branched off of her branch

**Review by DrDano — COMMENTED** [review]
I went through and added comments to all changes, this is the work of 2 people so there are quite a few, but I don't think any of this will break your code.

**Review by JosephJamesCoop — APPROVED** [review]

**Pull request merged**